app/controllers/event_attendances_controller.rb

```diff
@@ -149,10 +103,34 @@ def auto_complete_for_person_fullname
   
   
   protected
+
+  def upsert
+    @attendee = @event.attendees.where(person_id: attendee_params[:person_id]).first_or_create
+
+    if @attendee.update_attributes(attendee_params)
+      flash[:notice] = "Saved"
+      status = :ok
+    else
+      flash[:error] = "There was an error saving the attendance. Please try again, or contact an administrator if the issue persists."
```

Comment:
@mattharris5 This isn't a very user-friendly error message, but I'm not sure how else we should handle a failure here.